### PR TITLE
Improve FAQ questions

### PR DESCRIPTION
**Pull request by Andre601**
I don't see the point of having a FAQ question specificly for/about Essentials and EssentialsX.

Most issues I saw so far regarding Essentials was mostly about the user not using EssentialsXChat (or any other chat plugin for that matter).
The most asked question is still "Why does LP not show prefix/suffix in x?" which __should__ be answered in this FAQ page.

### CODE REVIEW

**Comment by Andre601** [comment]
Added the FAQ question if LuckPerms supports MC x since this question gets asked so many times, it starts to become annoying.

File: FAQ.md [hunk 1]
```diff
@@ -17,6 +22,10 @@ Such as:
...
 Please see [here](https://github.com/lucko/LuckPerms/wiki/Storage-system-errors).
 
-### I use SpongeForge and when I install LuckPerms do plugins no longer work
+### I use SpongeForge and when I install LuckPerms do plugins no longer work for OPs
 This is an intentional behaviour of SpongeForge. As soon as a permission plugin, in this case LuckPerms, is installed will it disable the OP system.  
 To fix this give the groups and users the required permissions for the plugins.
+
+### What versions does LuckPerms support?
+The default LuckPerms supports versions from 1.8.8 up to the latest release which is currently 1.15.1.  
+For the version 1.7.10 will you need to use the Legacy version of LuckPerms, which can be found on [Jenkins](https://ci.lucko.me/view/LuckPerms/job/LuckPerms/)
```
Comment (Tobi406) [inline FAQ.md]: Shouldn't a sentence end with a period?

**Review by Tobi406 — CHANGES_REQUESTED** [review]

File: FAQ.md (hunk 1 above)
Comment (Andre601) [reply FAQ.md]: You couldn't find a more minor nitpick, do you?

**Review by Tobi406 — APPROVED** [review]
No, I couldn't

**Comment by lucko** [comment]
Thanks :)

**Pull request merged**